services/google_scraping_service/db_updater.go

```diff
@@ -0,0 +1,53 @@
+package google_scraping_service
```

Comment:
As discussed before, I recommend to renamed this service to `google_search_service`. It sounds better (NO `-ing`) and it's more flexible if new Search services are added.

Reply:
Fixed in c31a4280dc801ab6a3c11aa295e54638900a9d8e